# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1422649712 18000
#      Fri Jan 30 15:28:32 2015 -0500
# Node ID 8fe29850fb74b00751cdf41da7ef5918e85b4ffc
# Parent  3b0b4d55002dba8dfdcc3db41712ed4655765cf2
don't insert packages when doing a simple search (bug #43769)

* ov-classdef.cc (cdef_manager::do_find_package_symbol): Call
find_package with load_if_not_found parameter set to false.

diff --git a/libinterp/octave-value/ov-classdef.cc b/libinterp/octave-value/ov-classdef.cc
--- a/libinterp/octave-value/ov-classdef.cc
+++ b/libinterp/octave-value/ov-classdef.cc
@@ -3949,17 +3949,17 @@ cdef_manager::do_find_package (const std
   return retval;
 }
 
 octave_function*
 cdef_manager::do_find_package_symbol (const std::string& pack_name)
 {
   octave_function* retval = 0;
 
-  cdef_package pack = find_package (pack_name, false);
+  cdef_package pack = find_package (pack_name, false, false);
 
   if (pack.ok ())
     retval = new octave_classdef_meta (pack);
 
   return retval;
 }
 
 //----------------------------------------------------------------------------
